components/locale/kmr_IQ.tsx

```diff
@@ -0,0 +1,46 @@
+import Pagination from 'rc-pagination/lib/locale/kmr_IQ';
+import DatePicker from '../date-picker/locale/kmr_IQ';
+import TimePicker from '../time-picker/locale/kmr_IQ';
+import Calendar from '../calendar/locale/kmr_IQ';
+import { Locale } from '../locale-provider';
+
+const localeValues: Locale = {
+  locale: 'ku',
```

Comment:
Shouldn't this be `kmr-iq`?

Reply:
Or `kmr`?

Reply:
@ahalimkara i agree it should be **kmr-iq** we would have **ckb-iq** also

Reply:
It is locale name of moment, I sorry there is no https://unpkg.com/moment@2.27.0/locale/kmr-iq.js now, but https://unpkg.com/moment@2.27.0/locale/ku.js .

Reply:
@afc163 this is a big issue -_- what exisits in momentjs for now is **ku** only which is **ckb-iq** for us @ahalimkara  what do you think can you add something like **kmr-iq.js** to momentjs ? ill add another one called **ckb-iq** as an alias for existing **ku**

Reply:
https://unpkg.com/moment@2.27.0/locale/ku.js is Sorani dialect so technically using `locale: 'ku'` is pointing to wrong momentjs translation.

I am not sure what happens when using a locale in antd which is not existing in momentjs, if it defaults to `en` then i think it is better to use English instead fo Sorani for Kurmanji dialect. Anyways, as this PR is merged, this might be discussion for another PR.